ui/mainframe.py

```diff
@@ -154,23 +154,23 @@ def __init__(self, mainUI=None):
         self.takeBackupBtn.configure(highlightcolor="black")
         self.takeBackupBtn.configure(pady="0")
         self.takeBackupBtn.configure(text='''Take Backup''')
-
-        userVar = tk.StringVar(mainUI, "User")
-        adminVar = tk.StringVar(mainUI, "Admin")
+        self.userVar = tk.StringVar(mainUI, "User")
```

Comment:
These variable shouldn't be a class variable so self is not needed here

Reply:
userVar is not required so please delete it.

Reply:
ok